src/SpectralDataDialog.h

```diff
@@ -0,0 +1,76 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  SpectralDataDialog.h
```

Comment:
This header file is actually not needed!   It is included only in one .cpp file, and that file influences the behavior of the program completely by means of registrations at static initialization time.  So the class definitions of this .h can simply move to the header-less .cpp file.

See all the files under src/menus -- they are headerless, but similarly manage to cause their items to appear in the toolbar menus.

Reply:
And once you do that, you could even put the whole file inside `namespace { ... }` which could have the tiny advantage of making fewer symbols in the linkage tables.  Maybe not important.